### ISSUE TITLE
Fine-grained permission control for Barcelona

### ISSUE THREAD

**Issue by davidsiaw** [degica/barcelona#546]
Currently all users who have access to Barcelona are either admins or developers. The lesser access level, developers while only barred from creation and deletion of districts, still have large and wide-ranging permissions across all projects.

We need a better way to determine per-project per-command permissions better.

**Comment by essa** [comment]
How about start using "Vault as an authentication/authorization backend"?
https://github.com/degica/barcelona/issues/401

**Comment by davidsiaw** [comment]
I'm not sure how vault can be used for this. It seemed from the ticket that more work needs to be done before vault can be used for it. But if its easier I'm all for it

**Comment by davidsiaw** [comment]
Closing in favor of vault for now
